tests/e2e/support/objects/url-navigation/actions.ts

```diff
@@ -30,12 +32,23 @@ export const navigateToDetailsPanelOfResource = async (
 }
 
 export const openResourceViaUrl = async (args: openResourceViaUrlArgs) => {
-  const { page, resource, user, space, editorName } = args
+  const { page, resource, user, space, editorName, client = '' } = args
   const fileId = await getTheFileIdOfSpaceFile(user, space, resource)
+  let fullUrl
 
-  const fullUrl = editorName
-    ? `${config.backendUrl}/external/open-with-web/?appName=${editorName}&fileId=${fileId}`
-    : `${config.backendUrl}/f/${fileId}`
+  switch (client) {
+    case 'desktop':
+      fullUrl = `${config.backendUrl}/external/open-with-web/?appName=${editorName}&fileId=${fileId}`
+      break
+    case 'mobile':
+      fullUrl = await getUrl({ user, fileId, editorName })
+      break
+    case '':
+      fullUrl = `${config.backendUrl}/f/${fileId}`
+      break
+    default:
+      throw new Error(`${client} not found`)
```

Comment:
should we use web url by default?
```suggestion
    default:
      fullUrl = `${config.backendUrl}/f/${fileId}`
```